code/controllers/subsystem/timer.dm

```diff
@@ -236,7 +236,7 @@ SUBSYSTEM_DEF(timer)
 	if (flags & TIMER_UNIQUE)
 		SStimer.hashes[hash] = src
 	if (flags & TIMER_STOPPABLE)
-		SStimer.timer_id_dict["timerid[id]"] = src
+		SStimer.timer_id_dict["timerid"+num2text(id,8)] = src
```

Comment:
There's a define for that

`TIMER_ID_NULL`

Its currently -1, and attempting to delete a timer that uses it triggers a stack_trace, addtimer manually returns it instead of the id right now.